Makefile

```
@@ -28,11 +28,11 @@ install-markdown-link-check:
 markdown-link-check:
 	@for f in $(ALL_DOCS); do $(MARKDOWN_LINK_CHECK) --quiet --config .markdown_link_check_config.json $$f; done
 
-.PHONY: install-markdown-lint
+.PHONY: install-markdownlint
 install-markdown-lint:
 	npm install -g markdownlint-cli
 
-.PHONY: markdown-lint
+.PHONY: markdownlint
 markdown-lint:
```

The PHONY no longer matches the actual target name now.

```suggestion
markdownlint:
```

same with install above

---

@bogdandrutu Why did you resolve this without fixing the issue?